### PR TITLE
Drop logical model mapping tests

### PR DESCRIPTION
**Pull request by isinyaaa**
Related: https://issues.redhat.com/browse/RHOAIENG-12015

Follow-up to https://github.com/kubeflow/model-registry/pull/295#pullrequestreview-2266606869.

## Merge criteria:

- All the commits have been [_signed-off_](https://github.com/kubeflow/community/tree/master/dco-signoff-hook#signing-off-commits)  (To pass the `DCO` check)

- [ ] The commits have meaningful messages; the author will squash them after approval or in case of manual merges will ask to merge with squash.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work.
- [ ] Code changes follow the [kubeflow contribution guidelines](https://www.kubeflow.org/docs/about/contributing/).

### CODE REVIEW

**Comment by isinyaaa** [comment]
cc: @tarilabs

File: .github/workflows/run-robot-tests.yaml [hunk 1]
```diff
@@ -56,9 +56,9 @@ jobs:
         # Run Robot Framework tests in REST mode from test/robot directory
         run: robot test/robot
       # Run Robot Framework tests in Python mode against running docker compose
-      - name: Run Robot Framework tests (Python mode)
-        # Run Robot Framework tests in Python mode from test/robot directory
-        run: TEST_MODE=Python robot test/robot/MRandLogicalModel.robot
+      # - name: Run Robot Framework tests (Python mode)
+      #   # Run Robot Framework tests in Python mode from test/robot directory
+      #   run: TEST_MODE=Python robot test/robot/MRandLogicalModel.robot
```
Comment (tarilabs) [inline .github/workflows/run-robot-tests.yaml:61]: Any reason for leaving this commented instead of just erasing them?

**Review by tarilabs — COMMENTED** [review]
Thank you, I fully support for this test removal now being an obsolete test, per link provided.

/lgtm

one small question please?

File: .github/workflows/run-robot-tests.yaml (hunk 1 above)
Comment (isinyaaa) [reply .github/workflows/run-robot-tests.yaml:61]: no reason actually, we can drop that too.

**Review by tarilabs — APPROVED** [review]
Thank you Isabella, approving per https://github.com/kubeflow/model-registry/pull/295#pullrequestreview-2266606869 and related discussions.

Also thanks @lampajr for chiming in

/lgtm
/approve

**Pull request merged**